Rakefile

```diff
@@ -83,7 +83,7 @@ task(:commits) do
   # populated with the range of commits the PR contains. If not available, this
   # falls back to `master..HEAD` as a next best bet as `master` is unlikely to
   # ever be absent.
-  commit_range = ENV['TRAVIS_COMMIT_RANGE'].nil? ? 'master..HEAD' : ENV['TRAVIS_COMMIT_RANGE']
+  commit_range = ENV['TRAVIS_COMMIT_RANGE'].nil? ? 'master..HEAD' : ENV['TRAVIS_COMMIT_RANGE'].sub(/\.\.\./, '..')
   puts "Checking commits..."
```

Comment:
Looks like this branch doesn't have the range output in the `puts` that was added to `stable` branch in #5365.

The output from TravisCI looks good at https://travis-ci.org/puppetlabs/puppet/jobs/180850536

```
Checking commits...
(PUP-6850) Exclude commits unrelated to a PR
...passed
```

Should we add it to this branch as part of this PR and deal with the merge conflict on the merge up?